limit.go

```diff
@@ -0,0 +1,75 @@
+package pilosa
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"time"
+)
+
+var (
+	ErrorTooManyConnections = errTooManyConnections{}
+)
+
+func NewBoundListener(maxActive int, l net.Listener) net.Listener {
```

Comment:
I don't think there's any rules here, but I would put `l net.Listener` as the first argument to the function.

Reply:
hmm...wonder what value adds.

Reply:
I don't think it adds any value. It just seems that logically if you're writing a wrapper you would structure it like `ObjWrapper(obj, arg1, arg2)`, and if you later added another argument to your Wrapper, you would just add it to the end of the list. But again, other than what amounts to me being picky, I don't think it adds any value.